packages/documentation-framework/app.js

```diff
@@ -107,8 +107,15 @@ const isPrerender = process.env.PRERENDER;
 // Don't use ReactDOM in SSR
 if (!isPrerender) {
   function render() {
-    const renderFn = isProd ? ReactDOM.hydrate : ReactDOM.render;
-    renderFn(<App />, document.getElementById('root'));
+    const container = document.getElementById('root');
+    // TODO: Enable StrictMode: https://reactjs.org/docs/strict-mode.html
```

Comment:
Enabling strict mode here seems to break the navigation, so I have omitted it in this PR. I've created an issue to deal with this at a later point (#3204).

Reply:
Yeah since PF is built with so many PF components, and there are a [number of issues](https://github.com/patternfly/patternfly-react/issues/7889) with strict mode in PF-react right now, it may be a time before we can enable strict mode in org.

Reply:
@nicolethoen note that this is React [Strict Mode](https://reactjs.org/docs/strict-mode.html), not TypeScript strict mode we're talking about.